boards/arc/axs/support/openocd.cfg

```diff
@@ -0,0 +1,80 @@
+#  Copyright (C) 2019 Synopsys, Inc.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+# Configure JTAG cable
+# SDP has built-in FT2232 chip, which is similar to Digilent HS-1, except that
+# it uses channgel B for JTAG, instead of channel A.
+interface ftdi
+
+# Only specify FTDI serial number if it is specified via
+# "set _ZEPHYR_BOARD_SERIAL 12345" before reading this script
+if { [info exists _ZEPHYR_BOARD_SERIAL] } {
+       ftdi_serial $_ZEPHYR_BOARD_SERIAL
+}
+
+ftdi_vid_pid 0x0403 0x6010
+ftdi_layout_init 0x0088 0x008b
+ftdi_channel 1
+
+
```

Comment:
Please allow specification of Degilent probe by its serial number, see https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arc/em_starterkit/support/openocd.cfg#L5 and 
 https://github.com/zephyrproject-rtos/zephyr/commit/946ed0e3695a0744f3b58fa741ec0204a8ac8d5f

Basically you need to add here:
```
# Only specify FTDI serial number if it is specified via
# "set _ZEPHYR_BOARD_SERIAL 12345" before reading this script
if { [info exists _ZEPHYR_BOARD_SERIAL] } {
       ftdi_serial $_ZEPHYR_BOARD_SERIAL
}
```